### ISSUE TITLE
[3.6.0] Adding new custom emoji or profile avatar leads to: The formData parameter MUST be an instance of the FormData class.

### ISSUE THREAD

**Issue by AmatorPhasma** [RocketChat/Rocket.Chat#18736]
### Description:

- Adding new custom emojis on the admin page and changing profile avatar doesn't work as expected.

### Steps to reproduce:

1. Go to Adminpage -> Custom Emoji.
2. Choos a name and a gif file.
3. Click on save

### Expected behavior:

New emoji is added.

### Actual behavior:

Get the err: The formData parameter MUST be an instance of the FormData class.
![001](https://user-images.githubusercontent.com/49910825/91646517-35da3e80-ea50-11ea-9cc3-a6d5a6b25101.png)

### Server Setup Information:

- Version of Rocket.Chat Server:  3.6.0
- Operating System: GNU/Debian 10 alias Buster
- Deployment Method: tar
- Number of Running Instances: 5
- DB Replicaset Oplog: Yes
- NodeJS Version: 12.16.1
- MongoDB Version: 4.0.20

### Client Setup Information

- Desktop App or Browser Version: Both (Browser Firefox/Opera)
- Operating System: Linux and Windows

### Relevant logs:

No Log entry on the Server

**Comment by JargeZ** [comment]
I was just about to write a report on this.
As far as I could gather information, it broke during the transition from version 3.5.3 to 3.5.4

**Comment by enzote84** [comment]
Hi!
I have the same problem. Changing avatar worked fine in 3.5.3, and this error started after updated to 3.6.0.

**Comment by prigal** [comment]
New comers to Rocket.Chat, fresh install on 3.6, same problem.

**Comment by danielmorlock** [comment]
Same here, 3.6.0

**Comment by MikhailJDobrynin** [comment]
the same case in 3.6.0

**Comment by akong77** [comment]
the same case in 3.6.0

**Comment by ayr-ton** [comment]
Also happening with me in 3.6.0

**Comment by docsolid** [comment]
same here

**Comment by solohsi** [comment]
same here, fresh 3.6.0, can't upload avatar, url method works well

**Comment by charlyrock** [comment]
Same here Version 3.6.0
If I try to change the avatar of a channel, the "hide system message" option is reset. 
And I get the following message: `Exception in setTimeout callback: TypeError: Cannot read property 'etag' of undefined     at app/lib/server/functions/setRoomAvatar.js:34:46     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)     at packages/meteor.js:550:25     at runWithEnvironment (packages/meteor.js:1286:24)`

Need urgent a solution to fix it

**Comment by cremfert** [comment]
Same here, upload not working, can only use avatar via url

**Comment by jtorrex** [comment]
Another case here, I just updated to the latest release and some users reported me this case.

**Comment by LeeThompson** [comment]
Same problem here.

Can't add a custom emoji (although ones already added work).
Tried .gif and .png


```
Version: 3.6.0
OS: Linux (3.10.105) (Official Docker Container)
Node Version: v12.16.1
MongoDB Version: 4.4.0 (wiredTiger)

```

**Comment by LeeThompson** [comment]
Over a week and no dev comments?

**Comment by Godman-quake** [comment]
me same problem

**Comment by plociu84** [comment]
same here

**Comment by erkapilmehta** [comment]
Is there any solution or workaround available for this issue? We are also facing this same issue with latest installation of rocketchat version 3.6

**Comment by bkraul** [comment]
Same here with develop e30af61.

**Comment by LeeThompson** [comment]
> Is there any solution or workaround available for this issue? We are also facing this same issue with latest installation of rocketchat version 3.6

There hasn't been _any_ dev acknowledgement of this issue.

**Comment by Atomzwieback** [comment]
Same Problem here would be cool if a Dev could take a look at it.

**Comment by mojitaleghani** [comment]
The same issue appears here, I have two different instance running..one is a docker-container mode (one container running behind a web-proxy) and a two-instance nodes running behind nat ,both version 3.6.0 . the cluster-mode one (docker-container) has the same issue :(

**Comment by nstoeckigt** [comment]
This only seems to effect User Avatars, I successfully managed to change a Channel Image.

**Comment by bkraul** [comment]
@nstoeckigt , it also affects custom emoji.

**Comment by kokomo123** [comment]
No dev comments yet. I'm just including myself in here because there hasn't been no note and issues keep piling up. I'm trying to be nice by being patient. So hopefully they'll be able to fix this, else im gonna have to recreate my docker image and go back to 3.5.3 which isn't bad.

**Comment by simonshivy** [comment]
I'm just upgraded to 3.6.0 and experienced same issue. No way to change user avatar, except changing image physically on disk.

**Comment by frdmn** [comment]
Sorry folks, for the lack of response. We just [released a fix](https://github.com/RocketChat/Rocket.Chat/pull/18841) for this which will be shipped with the next patch release, 3.6.1.

**Comment by RaelZero** [comment]
> Sorry folks, for the lack of response. We just [opened a PR](https://github.com/RocketChat/Rocket.Chat/pull/18841) to address this, which will be probably shipped with the next patch release, 3.6.1.

Thanks for the fix :)

I think people are feeling a bit too entitled around here - you're all doing a great job. Personally, I'm very grateful :D

**Comment by Atomzwieback** [comment]
> Sorry folks, for the lack of response. We just [opened a PR](https://github.com/RocketChat/Rocket.Chat/pull/18841) to address this, which will be probably shipped with the next patch release, 3.6.1.

Thank yo very much, this fix is very appreciated. When does the release of 3.6.1 is aimed at?

**Comment by frdmn** [comment]
@Atomzwieback It should land today or tomorrow I think 🤞

**Comment by LeeThompson** [comment]
Awesome, glad to hear this being addressed!

**Comment by Atomzwieback** [comment]
> @Atomzwieback It should land today or tomorrow I think 🤞

Docker container release should be shortly after the normal release right?

**Comment by frdmn** [comment]
> Docker container release should be shortly after the normal release right?

Yes, those are built automatically whenever theres a new (regular) release appearing in the [GitHub releases section](https://github.com/RocketChat/Rocket.Chat/releases).

**Comment by ankar84** [comment]
> Sorry folks, for the lack of response. We just [opened a PR](https://github.com/RocketChat/Rocket.Chat/pull/18841) to address this, which will be probably shipped with the next patch release, 3.6.1.

Great news, Jonas!
And please don't forget to prepare bug fixes for that bugs (I think part of them could be fix by https://github.com/RocketChat/Rocket.Chat/pull/18841)
https://github.com/RocketChat/Rocket.Chat/issues/18831 and https://github.com/RocketChat/Rocket.Chat/issues/18737
https://github.com/RocketChat/Rocket.Chat/issues/18829 and https://github.com/RocketChat/Rocket.Chat/issues/18740

**Comment by kokomo123** [comment]
Thank you to the devs personally from the bottom of my heart. I hope you guys been okay. You are awesome in your own right. thank you so much for maintaining this beautiful chat. :) 💯 ❤️ 👍🏻 😎

**Comment by Aarkon** [comment]
Can confirm on 3.5.3.

**Comment by NineBAR** [comment]
I'm still seeing this error in 3.6.0.

**Comment by LeeThompson** [comment]
@NineBAR It was reported in 3.6.0.   It is supposedly fixed for 3.6.1 (obviously will confirm once it's released).

**Comment by NineBAR** [comment]
> 
> 
> @NineBAR It was reported in 3.6.0. It is supposedly fixed for 3.6.1 (obviously will confirm once it's released).

My mistake. Thanks!
